### PR TITLE
Enabled symengine in gems.yml

### PR DESCRIPTION
**Pull request by abinashmeher999**
[symengine.rb](https://github.com/symengine/symengine.rb) is available on rubygems.org as a gem and can be installed by `gem install symengine`. This is required for https://github.com/SciRuby/sciruby-notebooks/pull/3

cc @v0dro

### CODE REVIEW

**Comment by abinashmeher999** [comment]
The build fails because `libsymengine`(SymEngine C++ library) is not installed before symengine gem gets installed.
```
CMake Error at CMakeLists.txt:7 (find_package):

  Could not find a configuration file for package SymEngine.

  Set SymEngine_DIR to the directory containing a CMake configuration file

  for SymEngine.  The file will have one of the following names:

    SymEngineConfig.cmake

    symengine-config.cmake

```
We need to configure travis to do that.

**Comment by v0dro** [comment]
is it possible to write a script that can pull the relevant libraries, compile them and put the binaries in the right place?

**Comment by abinashmeher999** [comment]
We can list the steps in travis. I will try to do it.

*Regards,*

*Abinash Meher*

**Comment by abinashmeher999** [comment]
@v0dro I have configured travis. The builds fail for another gem. Please have a look.

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,11 @@ matrix:
     - rvm: ruby-head
 
 before_install:
+  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
-  - sudo apt-get install -qq gnuplot-nox libgsl0-dev libsqlite3-dev libatlas-base-dev libmagickcore-dev
+  - sudo apt-get install -qq gnuplot-nox libgsl0-dev libsqlite3-dev libatlas-base-dev libmagickcore-dev cmake libgmp-dev g++-4.7
   - sudo ln -s /usr/lib/libtoolize /usr/lib/libtool
+  - source scripts/install_symengine.sh
```
Comment (isuruf) [inline .travis.yml]: Don't do source here to prevent variables set in the script affecting other gems.

**Comment by v0dro** [comment]
The failure is because of iruby, which is normal.

@minad this looks good. Your thoughts?

**Comment by rajithv** [comment]
ping

**Comment by abinashmeher999** [comment]
I thought this was merged. Is there anything else that needs to be done?

**Comment by v0dro** [comment]
Looks good to me. I had merged your notebook, Abinash. @minad  can you merge this?

**Comment by minad** [comment]
No, we won't merge this like that, since we cannot clutter this gem with special installation scripts.
This should be fixed on the gem level (inside symengine.rb), like for example nokogiri does it with libxml.

**Comment by minad** [comment]
And sorry guys for intervening that late, but this must be fixed differently. Otherwise we pay a much higher maintenance price for the sciruby gem in the long run.

**Comment by isuruf** [comment]
You mean, install symengine inside symengine.rb? Are you open to using a compiler with C++11 features like g++-4.8?

**Comment by minad** [comment]
Yes that should be ok.

Am 3. Mai 2016 06:29:03 MESZ, schrieb Isuru Fernando <notifications@github.com>:

**Pull request closed without merging**